nemo/lightning/data.py

```diff
@@ -19,6 +19,7 @@
 from typing import List, Literal, Optional
 
 import torch
+from lightning.pytorch.overrides.distributed import _IndexBatchSamplerWrapper
```

Comment:
please change this to 
`from pytorch_lightning.overrides.distributed import _IndexBatchSamplerWrapper
`

Reply:
changed it here: [7a81b01](https://github.com/NVIDIA/NeMo/pull/10934/commits/7a81b01ad39bbcb4181c18cf5b3c50de25b073cb)

Reply:
thanks, running ci again